reflex/model.py

```diff
@@ -300,7 +382,6 @@ def migrate(cls, autogenerate: bool = False) -> bool | None:
         return True
 
     @classmethod
-    @property
```

Comment:
i think this might be a breaking change... my example app [rx_shout](https://github.com/masenf/rx_shout/tree/main) was using `select` as a class property on a model.

And now it throws stack at runtime

```python
Traceback (most recent call last):
  File "/Users/masenf/code/reflex-dev/reflex/reflex/state.py", line 1454, in _process_event
    events = fn(**payload)
             ^^^^^^^^^^^^^
  File "/Users/masenf/code/reflex-dev/repro-datetime-model/rx_shout/state.py", line 48, in on_load
    self.load_entries()
  File "/Users/masenf/code/reflex-dev/repro-datetime-model/rx_shout/state.py", line 45, in load_entries
    self.entries = session.exec(Entry.select.order_by(Entry.ts.desc())).all()
                                ^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'function' object has no attribute 'order_by'
```

Workaround is to call the function instead of treating it like a property: `Entry.select().order_by(...)`

Reply:
Yes, that's a breaking change. However, I still think it was the right decision to drop those deprecated decorator chains.